packages/db/tests/lib/test-execution.js

```diff
@@ -83,9 +83,25 @@ const newGranuleAssociatedWithExecution = async (
   return { executionCumulusId, granuleCumulusId };
 };
 
-test.before(async (t) => {
+const batchCreateExecutions = async (knex, cumulusCollectionId, length = 100) => {
+  const executionPgModel = new ExecutionPgModel();
+  const promiseArray = Array.from(
+    { length },
+    () => executionPgModel.create(
+      knex,
+      fakeExecutionRecordFactory({
+        timestamp: new Date(Date.now()),
+        collection_cumulus_id: cumulusCollectionId,
+      })
+    )
+  );
+  return await Promise.all(promiseArray);
+};
+
+test.beforeEach(async (t) => {
+  t.context.testDbName = `execution_${cryptoRandomString({ length: 10 })}`;
```

Comment:
This is cleaner than how we'd been doing this

Reply:
It's worth pointing out that this *does* make the test slightly slower.